CryptoTechReminderSystem.AcceptanceTest/HarvestTimeEntriesExampleResponse.json

```diff
@@ -55,5 +55,17 @@
       "created_at":"2019-03-01T10:08:00Z",
       "updated_at":"2019-03-01T10:08:00Z"
     }
-  ]
+  ],
+  "per_page":100,
+  "total_pages":1,
+  "total_entries":6,
+  "next_page":null,
+  "previous_page":null,
+  "page":1,
+  "links":{
```

Comment:
Do we need `links` at all? I think we can safely remove it and keep only the son subset we need.

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/33/commits/8b1e7d187d0117c1d01e68d48b8a86dc78219a36.